src/codegeneration/ClassTransformer.js

```diff
@@ -137,18 +136,19 @@ function classMethodDebugName(className, methodName, isStatic) {
   return createBindingIdentifier('$__' + className + '_prototype_' + methodName);
 }
 
-export class ClassTransformer extends TempVarTransformer{
+export class ClassTransformer extends TempVarTransformer {
   /**
    * @param {UniqueIdentifierGenerator} identifierGenerator
    * @param {ErrorReporter} reporter
    * @param {boolean} showDebugNames options.debugNames
    */
-  constructor(identifierGenerator, reporter, debugNames) {
+  constructor(identifierGenerator, reporter, options) {
     super(identifierGenerator);
```

Comment:
After further reviewing I don't think would be good. ParseTreeTransformer does not care about these params so it should not need to manage them.